phpBB/styles/prosilver/theme/icons.css

```diff
@@ -85,3 +85,26 @@
     line-height: .9;
     height: 12px;
 }
+
+
+blockquote cite:before, .uncited:before {
+	-webkit-font-smoothing: antialiased;
+	-mos-osx-font-smoothing: grayscale;
+	font-family: FontAwesome;
+	font-size: 14px;
+	font-weight: normal;
+	font-style: normal;
+	font-variant: normal;
+	line-height: 1;
+	text-rendering: auto;
+	display: inline-block;
+	padding-right: 5px;
+	content: '\f10d'; /* Font Awesome quote-left */
+}
+
+.rtl blockquote cite:before, .rtl .uncited:before {
+	padding-right: 0;
+	padding-left: 5px;
+	content: '\f10e'; /* Font Awesome quote-right */
```

Comment:
my intent was not to confuse so sorry. The reason i suggested to move `content` only is for two reasons.

1. because it was split to begin with. The `content` props are replacing the `background-image` props. essentially everything from colours.css should move to icons.css and everything from rtl should remain in rtl as it was since this is for 3.2.x and we want to maintain as much backwards compatibility as possible by keeping changes as small as possible. Moving things around between files is usually something we try to avoid to keep things simple and sane. It might not have any ramifications to me or you but it may for end-users and theme authors who have made changes to the base theme.

2. Because in 3.3 we will be replacing all of the font-awsome stuff with svg icons for better compatibility/flexibility so i am pre-staging for that as icons.css will be completely overhauled.